inactivity.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: Maintainer and Repository Inactivity
+nav_order: 6
+---
+[//]: # (SPDX-License-Identifier: CC-BY-4.0)
+
+# Inactivity Policies
+
+As part of the normal lifecycle of a project codebases and maintainers come and go. This document formalizes a Hyperledger wide policy for moving maintainers to emeritus status when a project does not have an explicit policy of their own, and the responsibilities of the Technical Steering Committee (TSC), Hyperledger Foundation staff, and maintainers in this process.
+
+## Maintainer Inactivity
+
+This policy applies to projects that do not have an explicit maintainer inactivity policy. Where a project has an established and functioning policy, only that project's policy will apply.
+
+Hyperledger very much appreciates the contributions of all maintainers but removing write privileges is in the interest of an orderly and secure project.
+
+Activity can be code contributions, code reviews, issue reporting, or any other such activity trackable by GitHub attributed to a Hyperledger repository.
+
+When a maintainer has not had any activity in a particular project for three months they will receive a notification informing them of the inactivity policies. The means and manner of notification (email, github mentions, etc.) will be at the discretion of the TSC Chair or who the TSC Chair designates. 
```

Comment:
I don't want to write the means of warning in to the policy.  This makes it flexible if a project changes chat servers again, or if one project uses email more than chat, or any other such eventualities.

Since this is a courtesy notice it may not come in the form of a PR.